README.md

```diff
@@ -1808,35 +1809,38 @@ A bit on how all this fits together:
 - Investment gains, such as buying and selling a stock, are similarly taxed at “ordinary”
   rates, unless they are [**long-term**](https://www.irs.gov/taxtopics/tc409.html), which means you
   held the asset for more than a year.
-- You also pay a number of other federal taxes (see a 📥[2018 summary for all states](https://www.adp.com/tools-and-resources/compliance-connection/state-taxes/2018-fast-wage-and-tax-facts.aspx)),
+- You also pay a number of other federal taxes (see a
+  [📥2018 summary for all states](https://www.adp.com/tools-and-resources/compliance-connection/state-taxes/2018-fast-wage-and-tax-facts.aspx)),
   notably:
   - **6.2%** for Social Security on your first $118,500
   - **1.45%** for Medicare
   - **0.9%**
     [Additional Medicare Tax](https://www.irs.gov/Businesses/Small-Businesses-&-Self-Employed/Questions-and-Answers-for-the-Additional-Medicare-Tax)
     on income over $200,000 (single) or $250,000 (married filing jointly)
   - **3.8%**
-    [Net Investment Income Tax (NII)](https://www.irs.gov/uac/Newsroom/Net-Investment-Income-Tax-FAQs)
-    ([enacted](https://www.taxpolicycenter.org/briefing-book/what-tax-changes-did-affordable-care-act-make)
-    as part of the Affordable Care Act,
-    [also called “Obamacare”](https://www.investopedia.com/articles/personal-finance/120513/how-obamacare-raising-your-taxes.asp))
+    [Net Investment Income Tax](https://www.irs.gov/uac/Newsroom/Net-Investment-Income-Tax-FAQs)
+    (NII) ([enacted](https://www.taxpolicycenter.org/briefing-book/what-tax-changes-did-affordable-care-act-make)
+    as part of the Affordable Care Act, also called
+    [“Obamacare”](https://www.investopedia.com/articles/personal-finance/120513/how-obamacare-raising-your-taxes.asp))
     on investment income if you make over $200,000 (single) or $250,000 (married filing
     jointly)
 - Ordinary federal income tax, Social Security, and Medicare taxes are withheld from your
   paycheck by your employer and are called
   [**employment taxes**](https://www.irs.gov/Businesses/Small-Businesses-&-Self-Employed/Understanding-Employment-Taxes).
-- 🔹 Long-term capital gains are
-  [taxed at a lower rate](https://www.fool.com/taxes/2017/12/11/long-term-capital-gains-tax-rates-in-2018.aspx)
+- 🔹 Long-term capital gains are taxed at a
+  [lower rate](https://www.fool.com/taxes/2017/12/11/long-term-capital-gains-tax-rates-in-2018.aspx)
   than ordinary income tax: **0%**, **15%**, or **20%**. This covers cases where you get
   dividends or sell stock after holding it a year.
   If you are in the middle brackets (more than about $37K and less than $413K of ordinary
-  income), your long-term capital gains rate is 15% ([more details](https://taxfoundation.org/2018-tax-brackets/)).
+  income), your long-term capital gains rate is 15%. You can find more detail on tax
```

Comment:
This is a longer conversation but the "pure citation" links are not working for me. They are too few and look like mistakes. And even if we were to use a "pure citation" format, we never want to (or want to avoid as much as possible) using "more details" or "click here to read more."

Reply:
It's a lot neater. Citing the tax found doesn't imply that it's original (eg we don't say, "it's 15%, according to the tax found"). We're linking it as a good resource if you need to know more about tax brackets. Stet.